tools/pythonpkg/src/pandas/bind.cpp

```diff
@@ -68,6 +68,9 @@ static LogicalType BindColumn(PandasBindColumn &column_p, PandasColumnBindData &
 			bind_data.numpy_type.type = NumpyNullableType::CATEGORY;
 			vector<string> enum_entries = py::cast<vector<string>>(categories);
 			idx_t size = enum_entries.size();
+			if (enum_entries.empty()) {
+				throw InvalidInputException("Could not create an ENUM from an empty Categorical");
```

Comment:
Perhaps we can fallback to `LogicalType::VARCHAR` here instead so we can query the dataframe?

Reply:
Maybe, but I don't really see when anyone would use this, creating a column out of an empty Categorical means it can't hold any values, the set of available options is empty

Reply:
```py
data = ['cat', 'dog', 'cat', 'bird', 'dog', 'cat']
categorical_column = pd.Categorical(data, categories=[])
df = pd.DataFrame({'category': categorical_column})
print(df)
```
```
  category
0      NaN
1      NaN
2      NaN
3      NaN
4      NaN
5      NaN
```

Reply:
I don't think anyone uses this intentionally - but at least one user seems to have encountered this error. Users might not explicitly make a pandas DataFrame without categories, but with no data the categories list might end up empty, causing a strange error on the DuckDB side when reading. If possible since Pandas accepts this, I think we should accept it as well.

Reply:
e.g. something like this seems plausible:

```py
import pandas as pd
import numpy as np

def create_categorical(data):
	categorical_column = pd.Categorical(data, categories=np.unique(data))
	df = pd.DataFrame({'category': categorical_column})
	return df

print(create_categorical([]))
-- Empty DataFrame
-- Columns: [category]
-- Index: []
```